### DISCUSSION TITLE
Updating JPEG and TIFF metadata for an image not reflected in File Explorer in Windows

### DISCUSSION THREAD

**Discussion by JeffP134** [SixLabors/ImageSharp#2503] [Q&A]
I am working in .NET 6 using Visual Studio 2022, with version 3.0.1 of the ImageSharp package.

I am building a tool to update the metadata on images that I am capturing and editing.  I have multiple related files that share most of the same metadata, so I am trying to avoid having to edit this information file by file.  

I can edit the metadata in the JPEG ExifProfile and TIFF ExifProfile just fine and save this information to the file system.  However, not all of the updated metadata tags are being reflected in Windows 10 File Explorer after the save.
- JPEG: Title (XPTitle, ImageDescription)
- TIFF: Title (XPTitle, ImageDescription), Date Taken (DateTimeOriginal)
I can also manually edit the title field in Windows 10, and the same changed metadata tags show up when I read the image into the editor again, so it looks like Windows and ImageSharp are using the same tags.

Am I missing something in the code to get the changed Exif tags to be picked up by Windows 10?  

The following screenshots show the sequence of changes.

**JPEG Image Data Prior to Editing using ImageSharp**
![Metadata Pre Edit in File Explorer](https://github.com/SixLabors/ImageSharp/assets/141293540/1319ca48-3364-4b39-aada-7e3c884afdf1)
**JPEG ExifProfile Prior to Editing using ImageSharp**
![ExifProfile Pre Edit in Visual Studio 2022](https://github.com/SixLabors/ImageSharp/assets/141293540/be8d1494-ae36-49b9-81b9-86c829600206)
**JPEG ExifProfile After Editing using ImageSharp**
![ExifProfile Post Edit in Visual Studio 2022](https://github.com/SixLabors/ImageSharp/assets/141293540/285c2e63-9d28-4a55-9a4f-ad55270df098)
**JPEG Image Data After Saving to Disk using ImageSharp**
Note that Title change is not reflected in File Explorer
![Metadata Post Edit in File Explorer](https://github.com/SixLabors/ImageSharp/assets/141293540/0cf06dc7-0976-4a16-94d6-4defab8fab83)
**JPEG ExifProfile in ImageSharp after Editing Title using File Explorer**
Note how Windows and ImageSharp seem to be using the same Exif tags.
![ExifProfile after Editing in File Explorer](https://github.com/SixLabors/ImageSharp/assets/141293540/bb3b2b12-d0d9-4b1d-bde6-5275ba63d6b7)

**Comment by JimBobSquarePants** [comment]
There’s a very strong chance windows is caching that data. I’d check with a third party [exif tool](https://exiftool.org/).
